source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -446,13 +446,19 @@
 								z-index: 0;
 								right: 0;
 
-								input[type="checkbox"]:checked ~ .checkbox-svg-wrapper span.closed-eye, input[type="checkbox"]:not(:checked) ~ .checkbox-svg-wrapper span.opened-eye {
+								input[type="checkbox"]:checked ~ .checkbox-svg-wrapper span.crossed-eye, input[type="checkbox"]:not(:checked) ~ .checkbox-svg-wrapper span.eye {
 									display: none;
 								}
 
-								svg {
+								span.eye, span.crossed-eye {
 									@include size(26px);
-									margin-left: 5px;
+
+									svg {
+										@include size(22px , 14px);
+										fill: #757575;
+										margin-left: 3px;
```

Comment:
Move these to the span.eye, span.crossed-eye. You might need to also add `display: block` to it or to this svg.